packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -131,6 +131,8 @@ class _CupertinoTextFieldSelectionGestureDetectorBuilder extends TextSelectionGe
 /// field (e.g., by pressing a button on the soft keyboard), the text field
 /// calls the [onSubmitted] callback.
 ///
+/// {@macro flutter.widgets.editableText.complexCharacters}
```

Comment:
What does this macro do?

Reply:
Elsewhere I defined a template with:

```
{@template flutter.widgets.editableText.complexCharacters}
```

So now every place that uses this macro will import the text defined in the template.

Reply:
Should this paragraph have the same   /// ## Handling emojis and other complex characters header as well?